app/retail/templates/mission.html

```diff
@@ -20,429 +20,430 @@
-      <div class="row mb-5">
-        <div class="col-md-12 d-flex align-items-center">
-          <img
-            class="icon"
-            src="{% static "v2/images/mission/believe/bounty.png" %}"
-          />
-          <span class="icon-text">
-          {% blocktrans %}
-           We believe in open standards and fat protocols, and have built Gitcoin with StandardBounties -- an Open, Free, Fair, Bounty Protocol on Ethereum.
-          {% endblocktrans %}
-          </span>
+    <div class="container-fluid bg-light" id="we_want">
+      <div class="row">
+        <div class="col pb-4">
+          <h4 class="text-left text-uppercase">
+            {% blocktrans %}
+            <span class="text-highlight-green">We</span> Want
+            {% endblocktrans %}
+          </h4>
         </div>
       </div>
 
-      <div class="row mb-5">
-        <div class="col-md-12 d-flex align-items-center">
-          <img
-            class="icon"
-            src="{% static "v2/images/mission/believe/no_token.png" %}"
-          />
-          <span class="icon-text">
-          {% blocktrans %}
-            We believe that an ICO and tokenization is not the best business model for our dapp. We dot not have a token
-          {% endblocktrans %}
-          </span>
+      <div class="row animatedParent we-want">
+        <div class="col-md-12 col-lg-8 col-xl-6 animated fadeInLeftShort">
+          <div class="row mb-5">
+            <div class="col-md-10 col-lg-8 d-flex align-items-center">
+              <img
+                class="icon"
+                src="{% static "v2/images/mission/want/rocket_man.svg" %}"
+              />
+              <span class="icon-text">
+              {% blocktrans %}
+                We want to build a world in which everyone has the <strong>financial
+                leverage to leave</strong> their job.
+              {% endblocktrans %}
+              </span>
+            </div>
+          </div>
+
+          <div class="row mb-5">
+            <div class="col-md-10 col-lg-8 d-flex align-items-center">
+              <img
+                class="icon"
+                src="{% static "v2/images/mission/want/find_work.svg" %}"
+              />
+              <span class="icon-text">
+              {% blocktrans %}
+                We want it to be as <strong>easy to find work in software</strong>, as it is for an
+                Uber driver to find a ride today.
+              {% endblocktrans %}
+              </span>
+            </div>
+          </div>
+
+          <div class="row mb-5">
+            <div class="col-md-10 col-lg-8 d-flex align-items-center">
+              <img
+                class="icon"
+                src="{% static "v2/images/mission/want/incentive.svg" %}"
+              />
+              <span class="icon-text">
+                {% trans "We want to build a network with <strong>aligned incentives</strong>, especially between capital &amp; labour." %}
```

Comment:
I think since there is a `<strong>` tag, you have to use `{% blocktrans %}` instead of `{% trans %}`.

Reply:
Just the explain, there are a few things in mission.html to do. I just reindented and fixed the head tag, removed inline CSS.  I think mission refactor should be in a separate branch to don't mix the purpose of this branch (vision) @SaptakS @thelostone-mc